apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -459,7 +465,9 @@ const EventTypePage = (props: EventTypeSetupProps & { allActiveWorkflows?: Workf
         destinationCalendar={destinationCalendar}
       />
     ),
-    availability: <EventAvailabilityTab eventType={eventType} isTeamEvent={!!team} />,
+    availability: (
+      <EventAvailabilityTab eventType={eventType} isTeamEvent={!!team} teamMembers={teamMembers} />
```

Comment:
The type is not defined for eventType.team.members and to avoid lint errors using any, TeamMembers under eventTypeSetupProps is used, which is used in other components as well and has the same required data.